src/VisualStudio.Roslyn.SDK/SyntaxVisualizer/Roslyn.SyntaxVisualizer.Extension/SyntaxVisualizerContainer.xaml.cs

```diff
@@ -78,7 +80,10 @@ internal void UpdateThemedColors()
             if (activeClassificationFormatMap != null && activeEditorFormatMap != null)
             {
                 var classificationTypeRegistryService = GetMefService<IClassificationTypeRegistryService>();
```

Comment:
I would prefer to use a `Required` helper here, since we know this service is always provided.

```suggestion
                var classificationTypeRegistryService = GetRequiredMefService<IClassificationTypeRegistryService>();
```